static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -0,0 +1,188 @@
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Card from 'sentry/components/card';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCommit, IconGithub, IconInfo, IconMail} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {MissingMember} from 'sentry/types';
+
+type Props = {
+  missingMembers: {integration: string; users: MissingMember[]};
+  onSendInvite: (email: string) => void;
+};
+
+export function InviteBanner({missingMembers, onSendInvite}: Props) {
+  // TODO(cathy): include snoozing, docs link
+
+  const cards = missingMembers?.users?.slice(0, 5).map(member => (
+    <MemberCard key={member.userId} data-test-id={`member-card-${member.userId}`}>
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <IconGithub size="sm" />
+          <StyledExternalLink href={`http://github.com/${member.userId}`}>
+            {tct('@[userId]', {userId: member.userId})}
+          </StyledExternalLink>
+        </MemberCardContentRow>
+        <MemberCardContentRow>
+          <IconCommit size="xs" />
+          {tct('[commitCount] Recent Commits', {commitCount: member.commitCount})}
+        </MemberCardContentRow>
+        <Subtitle>{member.email}</Subtitle>
+      </MemberCardContent>
+      <Button
+        size="sm"
+        onClick={() => onSendInvite(member.email)}
+        data-test-id="invite-missing-member"
+        icon={<IconMail />}
+      >
+        {t('Invite')}
+      </Button>
+    </MemberCard>
+  ));
+
+  cards?.push(
+    <MemberCard key="see-more" data-test-id="see-more-card">
+      <MemberCardContent>
+        <MemberCardContentRow>
+          <SeeMoreContainer>
```

Comment:
Does this need to be a container? Wouldn't it be more simple just to make this a styled `p`?

Reply:
it's a styled div that's called SeeMoreContainer. we can drop the container bit in the name though

Reply:
No I meant like why does it need to be a div - why not just style a `p` element directly

Reply:
p adds extra margin to the top and bottom which is not what i want here